bloodytools/simulation_objects/simulation_objects.py

```diff
@@ -653,19 +655,16 @@ def simulate(self) -> bool:
             .format(self.filename)
           )
         else:
-          self.filename = str(uuid.uuid4()) + ".simc"
+          # temporary file names
+          self.uuid = str(uuid.uuid4())
+          self.filename = "{}.simc".format(self.uuid)
+          json_filename = "{}.json".format(self.uuid)
 
-          # if somehow the random naming function created the same name twice
-          if os.path.isfile(self.filename):
-            self.logger.info(
-              "Somehow the random filename generator generated a name ('{}') that is already on use.".format(
-                self.filename
-              )
-            )
-            self.filename = str(uuid.uuid4()) + ".simc"
           # write arguments to file
           with open(self.filename, "w") as f:
             # write the equal values to file
+            f.write("json={}\n".format(json_filename))
```

Comment:
Aren't we using `json2=` in SimulationCraft now?

Reply:
I think there is no difference between the 2.
https://github.com/simulationcraft/simc/blob/bfa-dev/engine/sim/sc_sim.cpp#L3378
looks like the old json report died.